### PR TITLE
Blazor 0.9.0 release

### PR DESCRIPTION
**Pull request by SteveSandersonMS**
**Do not merge**

This PR is based on `release/3.0-preview3` only so we can easily see the diffs. It only exists as a PR for convenient collaboration, not because it should later get merged. The intent is to build the Blazor 0.9.0 packages from this branch, then move the commits we intend to retain back to `master`.

### CODE REVIEW

File: src/Components/Blazor/Directory.Build.props [hunk 1]
```diff
@@ -2,9 +2,23 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory)..\, Directory.Build.props))\Directory.Build.props" />
 
   <PropertyGroup>
-    <ComponentsPackageVersion>$(PackageVersion)</ComponentsPackageVersion>
+    <!-- Out-of-band release -->
+    <UseProjectReferences>false</UseProjectReferences>
+    <ComponentsPackageVersion>3.0.0-preview3-19153-02</ComponentsPackageVersion>
```
Comment (SteveSandersonMS) [inline src/Components/Blazor/Directory.Build.props]: **TODO:** Determine the final `ComponentsPackageVersion`. It needs to be whatever ships as Preview 3.

**Comment by anurse** [comment]
Helix failure is because this PR is using an old version of `eng/helix/vstest/runtests.cmd`. I thought AzDO was supposed to run on the "merged" version of the PR but maybe not... @SteveSandersonMS can you rebase on `master`?

**Comment by SteveSandersonMS** [comment]
> can you rebase on master?

No, sorry! This branch is for building the binaries that will ship alongside Preview 3, so it needs to remain based on `release/3.0-preview3`.

You can ignore the helix failures for this PR. It's not going to be merged anyway. I'll make a separate PR for the changes that will actually get merged back to `master`.

**Comment by anurse** [comment]
Ah, that explains the problem, never mind then :). Carry on good sir

File: src/Components/Blazor/Blazor/src/Hosting/WebAssemblyHostExtensions.cs [hunk 2]
```diff
@@ -19,7 +21,16 @@ public static class WebAssemblyHostExtensions
...
+            // Behave like async void, because we don't yet support async-main properly on WebAssembly.
+            // However, don't actualy make this method async, because we rely on startup being synchronous
+            // for things like attaching navigation event handlers.
+            host.StartAsync().ContinueWith(task =>
+            {
+                if (task.Exception != null)
+                {
+                    Console.WriteLine(task.Exception);
+                }
+            });
```
Comment (rynowak) [inline src/Components/Blazor/Blazor/src/Hosting/WebAssemblyHostExtensions.cs]: This seems like a fine fix for the wasm case.

**Pull request closed without merging**